e2e/integration/ssitests.js

```diff
@@ -1,24 +1,55 @@
 import { apps } from "../helper/apps.list";
+import Sidebar from "../pageObject/Sidebar"
 context( 'Test for SSI', () => {
-
+    const sidebar = new Sidebar();
     before( () => {
-        cy.viewport(Cypress.env('width'), Cypress.env('height') );
-        cy.visit( Cypress.env( 'QA_HOST' ));
+        cy.viewport( Cypress.env( 'width' ), Cypress.env( 'height' ) );
+        cy.visit( Cypress.env( 'STAGE_HOST' ) );
         cy.login( Cypress.env( 'USERNAME' ), Cypress.env( 'PASSWORD' ) );
     } );
 
     it( 'Test for Notifications section', () => {
-        cy.get( 'slot[name="opc-nav-btn"] > button:first-child' ).click({force:true}).then( () => {
+        cy.wait(3000)
+        cy.get( 'slot[name="opc-nav-btn"] > button:first-child' ).click( { force: true } ).then( () => {
+
+            cy.get( '.opc-notification-drawer__header-title', { force: true } ).invoke( 'text' ).then( ( text ) => {
+                expect( text.trim() ).equal( 'Notifications' );
+            } );
+
+
+        } );
+        cy.get( ".opc-notification-drawer__header > button" ).click( { force: true } );
+    } );
+    it( 'Test Application Search in Side Bar', () => {
+        cy.get( 'slot[name="opc-nav-btn"] > button:last-child' ).focus().click( { force: true } ).then( () => {
+cy.wait(2000)
+            cy.get( '.opc-menu-drawer' ).within( () => {
 
-            cy.get( '.opc-notification-drawer__header-title',{force:true} ).invoke( 'text' ).then( ( text ) => {
-                expect( text.trim()).equal('Notifications' )
+                cy.get( 'input[name="seach"]' ).focus().type( "dsal" ,{force:true});
+                sidebar.getApplicationSearchResults().contains('DSAL' ).should( 'be.visible' );
+                cy.wait( 3000 )
+                cy.get( '.opc-menu-drawer__header' ).within( () => {
+                    cy.get('button').click({force:true})
                 })
 
 
+            } );
+
+        } );
+    });
+    it( 'Test Built in Services Search', () => {
```

Comment:
Just a doubt here, isn't this test same as just above one

Reply:
@akhilmhdh , there were 2 sections that were coming up. So, I wrote 2 different testcases, One should display result in Apps section , other 1 should display in Built in service section

Reply:
Also @akhilmhdh , then why two different sections have been created ? Applicaitons and built in services ?

Reply:
I wont recommend like that. The issue is its too dependent as deepesh said. Like when the app changes from one type of another test also causes  failure.

Reply:
> Also @akhilmhdh , then why two different sections have been created ? Applicaitons and built in services ?

Because there exist two section in one platform apps

Reply:
@akhilmhdh , my test data is coming from the data which I am seeing on the UI. So it will work across all the environments now

Reply:
https://github.com/1-Platform/one-platform/pull/1523/commits/801d90cc8905d15a2a9c057b2c62cd1817c67276

Reply:
Exactly. And it may not be same in each env. This may cause unnecessary failed test.